i18n/extra/pt-br.json

```diff
@@ -0,0 +1,236 @@
+{
+    "@metadata": {
+        "authors": []
+    },
+    "fallback_language": "en",
```

Comment:
I am not an expert on Portuguese however it may be better to define "pt" as the fall-back language rather than English.